### PR TITLE
stakedrop-demo material

### PR DESCRIPTION
**Pull request by sthompson22**
Material used for the stakedrop-demo.  Includes accounts, files, and documentation that describes the steps taken.

### CODE REVIEW

**Comment by sthompson22** [comment]
@nkuba @pdyraga Here is the material I used for the stakedrop demo.  I gave folks the link to the PR but having this merged soonish would  be helpful.

File: infrastructure/stakedrop-demo/activity-log.adoc [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+. **Start keep-ecdsa-client**
+.. cd `/stakedrop-demo`
+.. Run:
+```
+export KEEP_ECDSA_OPERATOR_ACCOUNT_PASSWORD=$(cat ./config/keep-ecdsa-operator-account-password.txt)
+export KEEP_ECDSA_CONFIG_DIR=$(pwd)/config
+export KEEP_ECDSA_PERSISTENCE_DIR=$(pwd)/persistence
+
+docker run \
```
Comment (changwu-tw) [inline infrastructure/stakedrop-demo/activity-log.adoc:93]: Would be better to have container running in background.

```
docker run -d \
--restart always \
```

**Comment by sthompson22** [comment]
For sure! - used foreground for fewer steps during the demo

File: infrastructure/stakedrop-demo/activity-log.adoc (hunk 1 above)
Comment (pdyraga) [reply infrastructure/stakedrop-demo/activity-log.adoc:93]: Let me merge it as-is and we'll follow-up in a separate PR, do not want to alter this work without Sloan's approval.

**Comment by sthompson22** [comment]
I've restricted access to the alchemy app checked-in here, now that the demo is over.

File: infrastructure/stakedrop-demo/activity-log.adoc (hunk 1 above)
Comment (sthompson22) [reply infrastructure/stakedrop-demo/activity-log.adoc:93]: For sure! - used foreground for fewer steps during the demo

**Pull request merged**